README.md

```diff
@@ -60,12 +72,12 @@ $ cargo build --release
 ## Usage
 
 In the following sections we'll be using excerpts from the
-[examples](./examples/rust-example). You can actually run the complete
+[C examples](./examples/c-example). You can actually run the complete
 example using Cargo from inside that directory.
```

Comment:
Cargo is the rust build tool, so this sentence isn't right. Should probably just be "You can actually run the complete example from inside that directory.", they can figure out how to build it. Or we can ask Jon for instructions to build it.

Reply:
:+1: Sure, we'll stick with the minimal edit for now:
You can actually run the complete example from inside that directory.